db/schema.rb

```diff
@@ -10,10 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_01_09_171633) do
+ActiveRecord::Schema[7.0].define(version: 2023_01_11_013539) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pg_stat_statements"
-  enable_extension "pgcrypto"
```

Comment:
Is this line removal expected?

Reply:
I rebuilt the database from scratch to run all the migrations and this resulted. I'll follow up.

Reply:
running the migration did restore this line which I've pushed